src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/ApplicationGatewayTests.ps1

```diff
@@ -97,102 +97,102 @@ function Test-ApplicationGatewayCRUD
 	try 
 	{
 		# Create the resource group
-		$resourceGroup = New-AzureRmResourceGroup -Name $rgname -Location $location -Tags @{ testtag = "APPGw tag"} 
+		$resourceGroup = New-AzResourceGroup -Name $rgname -Location $location -Tags @{ testtag = "APPGw tag"} 
```

Comment:
You should not make changes for existing cmdlets - only new cmdlets need to use the Az prefix